Python-path-traversal-files-main/AliVard/CDC-Early-Stopping/cv.py

```
@@ -0,0 +1,141 @@
+import numpy as np
+import pandas as pd
+import pickle
+import os
+from scipy.spatial.distance import cosine
+
+import sys
+
+from GeneralML import *
+
+if __name__ == '__main__':
+    flag_lr = eval(sys.argv[1])
```

Ruff (0.13.3)

Use of possibly insecure function; consider using `ast.literal_eval`

(S307)